test/config/config.go

```diff
@@ -46,15 +46,24 @@ const (
 	HTTP_GW_HOST                    = "localhost:20010"
 	DEVICE_PROVIDER                 = "plgd"
 	OPENTELEMETRY_COLLECTOR_HOST    = "localhost:55690"
+	COAPS_SCHEME                    = "coaps://"
+	COAPS_TCP_SCHEME                = "coaps+tcp://"
 )
 
 var (
-	CA_POOL     = os.Getenv("LISTEN_FILE_CA_POOL")
-	KEY_FILE    = os.Getenv("LISTEN_FILE_CERT_DIR_PATH") + "/" + os.Getenv("LISTEN_FILE_CERT_KEY_NAME")
-	CERT_FILE   = os.Getenv("LISTEN_FILE_CERT_DIR_PATH") + "/" + os.Getenv("LISTEN_FILE_CERT_NAME")
-	MONGODB_URI = "mongodb://localhost:27017"
-	NATS_URL    = "nats://localhost:4222"
-	OWNER_CLAIM = "sub"
+	CA_POOL                  = os.Getenv("LISTEN_FILE_CA_POOL")
+	KEY_FILE                 = os.Getenv("LISTEN_FILE_CERT_DIR_PATH") + "/" + os.Getenv("LISTEN_FILE_CERT_KEY_NAME")
+	CERT_FILE                = os.Getenv("LISTEN_FILE_CERT_DIR_PATH") + "/" + os.Getenv("LISTEN_FILE_CERT_NAME")
+	MONGODB_URI              = "mongodb://localhost:27017"
+	NATS_URL                 = "nats://localhost:4222"
+	OWNER_CLAIM              = "sub"
+	COAP_GATEWAY_UDP_ENABLED = os.Getenv("TEST_COAP_GATEWAY_UDP_ENABLED") == "true"
+	USE_COAP_SCHEME          = func() string {
```

Comment:
USE_COAP_SCHEME -> použi coap schému, to je naschvál? Čakal by som skôr USED_COAP_SCHEME (= použitá copa schéma) alebo možno ACTIVE_COAP_SCHEME (=aktívna copa schéma).

Reply:
Fixed